### ISSUE TITLE
error after update

### ISSUE THREAD

**Issue by nagyrobi** [travisghansen/hass-pfsense#84]
After today's update to 0.5.1 at HA restart:

```
2022-07-06 20:26:23 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.pfsense.update
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 618, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 635, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/pfsense/update.py", line 93, in <module>
    class PfSenseFirmwareUpdatesAvailableUpdate(PfSenseUpdate):
  File "/config/custom_components/pfsense/update.py", line 107, in PfSenseFirmwareUpdatesAvailableUpdate
    def installed_version(self) -> str | None:
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
2022-07-06 20:26:23 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform pfsense.update: Platform not found (Exception importing custom_components.pfsense.update).
2022-07-06
```

**Comment by Qpernicus** [comment]
I have the same error.

**Comment by nagyrobi** [comment]
Seems it's fixed by upgrading to HA 2022.07..?

**Comment by travisghansen** [comment]
Actually any chance you can test `next` branch?

**Comment by travisghansen** [comment]
Also in `main` and `v0.5.2` now..

**Comment by megapearl** [comment]
Same problem here:

`This error originated from a custom integration.

Logger: homeassistant.loader
Source: custom_components/pfsense/update.py:149
Integration: pfSense (documentation, issues)
First occurred: 23:52:29 (1 occurrences)
Last logged: 23:52:29

Unexpected exception importing platform custom_components.pfsense.update
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/loader.py", line 618, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/loader.py", line 635, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/homeassistant/.homeassistant/custom_components/pfsense/update.py", line 92, in <module>
    class PfSenseFirmwareUpdatesAvailableUpdate(PfSenseUpdate):
  File "/home/homeassistant/.homeassistant/custom_components/pfsense/update.py", line 149, in PfSenseFirmwareUpdatesAvailableUpdate
    self, version: str | None, backup: bool, **kwargs: Any
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
`

**Comment by travisghansen** [comment]
@megapearl have you updated to `v0.5.2`?

**Comment by megapearl** [comment]
Yes, as I mentioned above, I tried 0.5.2 too.

**Comment by GSzabados** [comment]
This is the error message in `v0.5.2`
```

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 618, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 635, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/config/custom_components/pfsense/update.py", line 92, in <module>
    class PfSenseFirmwareUpdatesAvailableUpdate(PfSenseUpdate):
  File "/config/custom_components/pfsense/update.py", line 149, in PfSenseFirmwareUpdatesAvailableUpdate
    self, version: str | None, backup: bool, **kwargs: Any
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

**Comment by megapearl** [comment]
Tried it on a different system too, same errors:

<html>
<body>


Version | core-2022.6.7
-- | --
Installation Type | Home Assistant Core
Development | false
Supervisor | false
Docker | false
User | homeassistant
Virtual Environment | true
Python Version | 3.9.13
Operating System Family | FreeBSD
Operating System Version | 13.1-RELEASE
CPU Architecture | amd64
Timezone | Europe/Amsterdam


</body>
</html>

This error originated from a custom integration.

Logger: homeassistant.loader
Source: custom_components/pfsense/update.py:149
Integration: pfSense (documentation, issues)
First occurred: 12:33:49 AM (1 occurrences)
Last logged: 12:33:49 AM

```
Unexpected exception importing platform custom_components.pfsense.update
Traceback (most recent call last):
  File "/usr/local/share/homeassistant/lib/python3.9/site-packages/homeassistant/loader.py", line 618, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/local/share/homeassistant/lib/python3.9/site-packages/homeassistant/loader.py", line 635, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/homeassistant/.homeassistant/custom_components/pfsense/update.py", line 92, in <module>
    class PfSenseFirmwareUpdatesAvailableUpdate(PfSenseUpdate):
  File "/home/homeassistant/.homeassistant/custom_components/pfsense/update.py", line 149, in PfSenseFirmwareUpdatesAvailableUpdate
    self, version: str | None, backup: bool, **kwargs: Any
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

**Comment by travisghansen** [comment]
Ah, I see 1 more place I missed it. I think I'll have a fix shortly..

**Comment by travisghansen** [comment]
Can someone test `v0.5.3`?

**Comment by GSzabados** [comment]
No, error! And the notification for a firmware update shows up as expected! Thanks!

**Comment by megapearl** [comment]
Fixed for me too, updated to v0.5.3 on 2 different systems, no error's anymore.
Thanks for your time and work! :-)
